src/Screens/Chat/chat.js

```diff
@@ -43,11 +44,24 @@ export default class Conversas extends Component {
       .collection("conversas")
       .doc(user)
 
-    TranslatorConfiguration.setConfig(
-      ProviderTypes.Google,
-      "AIzaSyC0j0BsAskqVIvaX2fcdvjsaw4fqGP5ut8",
-      "en"
-    )
+    firebase
```

Comment:
Acredito que seria melhor fazer a consulta pela linguagem do destinatario quando for enviar uma mensagem, pra evitar erros de atualização. 
Também não entendi porque fazer um forEach entre todos os usuarios pois o usuario pode ser acessado por sua key.